spec/modules/obfuscate_ids_spec.rb

```diff
@@ -3,25 +3,32 @@
 require "spec_helper"
 
 describe "ObfuscateIds" do
-  before do
-    @purchase = create(:purchase)
```

Comment:
We don't need to create a factory for this since we test those methods against mere numbers